templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Do You want to delete the driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
+    <input type="submit" value="Cancel" class="btn btn-primary">
```

Comment:
The Cancel button should lead to the previous page the user was on. see checklist